src/runtime/Makefile

```diff
@@ -216,6 +216,11 @@ DEFBINDMOUNTS := []
 # Image Service Offload
 DEFSERVICEOFFLOAD ?= false
 
+# SEV Guest Pre-Attestation
+DEFGUESTPREATTESTATIONSECRETGUID ?= 0a46e24d-478c-4eb1-8696-113eeec3aa99
```

Comment:
This is the wrong GUID. I'm not really sure how it got in here but we should be using `e6f5a162-d67f-4750-a67c-5d065f2a9910`